### PR TITLE
Rename OpenTelemetry Service to Collector in docs and comments

### PR DESCRIPTION
**Pull request by tigrannajaryan**
This begins implementation of https://github.com/open-telemetry/opentelemetry-service/issues/352

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ exporting will resume.
 
 ## <a name="config"></a>Configuration
 
-The OpenTelemetry Service (both the Agent and Collector) is configured via a
+The OpenTelemetry Collector (both the Agent and Collector) is configured via a
```
Comment (songy23) [inline README.md]:
```suggestion
The OpenTelemetry Collector is configured via a
```

File: README.md [hunk 2]
```diff
@@ -177,7 +177,7 @@ receivers:
 
 An exporter is how you send data to one or more backends/destinations. One or
 more exporters can be configured. By default, no exporters are configured on
-the OpenTelemetry Service (either the Agent or Collector).
+the OpenTelemetry Collector (either the Agent or Collector).
```
Comment (songy23) [inline README.md]:
```suggestion
the OpenTelemetry Collector.
```

File: README.md (hunk 1 above)
Comment (tigrannajaryan) [reply README.md]: Fixed, thank you.

File: README.md (hunk 2 above)
Comment (tigrannajaryan) [reply README.md]: Fixed.

**Comment by tigrannajaryan** [comment]
@owais @pjanotti @flands @bogdandrutu please review.

**Pull request merged**